pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -353,3 +480,83 @@ func (c *K8sOrchestrator) IsFSSEnabled(ctx context.Context, featureName string)
 	log.Debugf("cluster flavor %q not recognised. Defaulting to false", c.clusterFlavor)
 	return false
 }
+
+// IsFakeAttachAllowed checks if the volume is eligible to be fake attached and returns a bool value
+func (c *K8sOrchestrator) IsFakeAttachAllowed(ctx context.Context, volumeID string, volumeManager cnsvolume.Manager) (bool, error) {
+	log := logger.GetLogger(ctx)
+	//Check pvc annotations
+	pvcAnn, err := c.getPVCAnnotations(ctx, volumeID)
+	if err != nil {
+		log.Errorf("failed to get pvc annotations")
+		return false, err
+	}
+
+	if val, found := pvcAnn[common.AnnIgnoreInaccessiblePV]; found && val == "yes" {
+		log.Infof("Found %s annotation on pvc for volume %s set to yes", common.AnnIgnoreInaccessiblePV, volumeID)
```

Comment:
Info -> Debug.

Maybe you can add one Info log if you are returning fake attach as true which could be useful in production env.

Reply:
Done.